data-model/SDP-sample-records/script_sample1.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "https://sinaimanuscripts.library.ucla.edu/script/1",
+  "label": "Serto (script)",
+  "associated_writing_systems": ["Syriac"],
+  "temporal_coverage": {
+    "date_human-readable": "post-7th c. - present",
+    "date_normalized": "0650/"
+  },
+  "geographic_coverage": "Typically used in West Syrian manuscripts",
+  "contributor": ["gkessel"]
+}
```

Comment:
Include ISO codes? What we call scripts are called 'typographic variants' (and writing systems are called 'script codes'). Cf. https://en.wikipedia.org/wiki/ISO_15924#Syrj

Need a field for references? e.g., to source the dates of the scripts? (FWIW I got these approximate dates from George Kiraz's recent Grammar [Gorgias Press 2012], vol. 1 p. 214ff.)